plugins/module_utils/ansiblemodule.py

```diff
@@ -0,0 +1,6 @@
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+
+from ansible.module_utils.basic import AnsibleModule  # noqa: F401
```

Comment:
This is not blocking, but I'm curious why we are doing this?

Reply:
This is the place where we will load the Turbo mode. I will use a conditional block to only enable it if we're running Py3.6+.